src/WebJobs.Script/Description/Workers/ScriptInvocationResult.cs

```diff
@@ -2,11 +2,17 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
+using System.Collections.Immutable;
 
 namespace Microsoft.Azure.WebJobs.Script.Description
 {
     public class ScriptInvocationResult
     {
+        public static readonly ScriptInvocationResult Success = new ScriptInvocationResult
```

Comment:
Not sure I understand why this is named as it is, or why it exists? - aren't outputs captured the prop on line 18 already? If you're trying to reflect state, could we use a bool or enum?